daemon/container_unix.go

```diff
@@ -640,7 +640,7 @@ func (container *Container) updateNetworkSettings(n libnetwork.Network, ep libne
 		return err
 	}
 
-	if container.hostConfig.NetworkMode == runconfig.NetworkMode("bridge") {
+	if container.hostConfig.NetworkMode == runconfig.NetworkMode("default") {
 		networkSettings.Bridge = container.daemon.config.Bridge.Iface
```

Comment:
@huikang criu should not try to restore network resources. The docker checkpoint/restore design should leave that to libnetwork. This is because criu should not assume anything about even an existence of a bridge. If the network to which the container is attached to is backed by a totally different driver (like ipvlan/macvlan) there may not even be a bridge or veth.